### PR TITLE
Unification: Just make the node plugins similar to the others

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,15 @@
-# Docker image for the Drone SFTP plugin
-
 FROM alpine:3.3
 
 RUN apk update && \
-    apk add nodejs && \
-    apk add git && \
```
Comment (athieriot) [inline Dockerfile]: I believe that some dependencies from NPM are pulling things from Git. That's why I had to put it there

Comment (tboerger) [inline Dockerfile]: As you can see at http://beta.drone.io/drone-plugins/drone-sftp/4 all dependencies are installing fine :)

Comment (athieriot) [inline Dockerfile]: Yeah but the image used for the build does have git installed: 

selection_066

Comment (tboerger) [inline Dockerfile]: Yes, you are right. I missed the currently not executed docker build which is based on that.

Comment (tboerger) [inline Dockerfile]: Fixed, installing git again :)

**Comment by athieriot** [comment]
LGTM

**Comment by tboerger** [comment]
LGTM

**Pull request merged**